zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -41,6 +42,13 @@ public static void main(String[] args) throws Exception {
       noteId = zClient.createNote(notePath);
       System.out.println("Created note: " + noteId);
 
+      String newNotePath = notePath + "_rename";
+      zClient.renameNote(noteId, newNotePath);
+
+      NoteResult renamedNoteResult = zClient.queryNoteResult(noteId);
+      Assert.assertEquals(newNotePath, renamedNoteResult.getNotePath());
```

Comment:
Sorry, my fault, `ZeppelinClientExample` is not unit test class, it is only for demo. So need assert here.
You should add unit test in `ZeppelinClientIntegrationTest`

Reply:
I have added the unit test in ZeppelinClientIntegrationTest, BTW, why put the unit test in modul "zeppelin-interpreter-integration"?

Reply:
Because zeppelin-client depends on zeppelin-server, that's why we put zeppelin-client module's test in `zeppelin-interpreter-integration` which includes the end-to-end integration test.